pkg/operator/credentialsrequest/credentialsrequest_controller.go

```diff
@@ -372,14 +373,14 @@ func (r *ReconcileCredentialsRequest) Reconcile(request reconcile.Request) (reco
 		metrics.MetricControllerReconcileTime.WithLabelValues(controllerName).Observe(dur.Seconds())
 	}()
 
-	_, operatorIsDisabled, conflict, err := utils.GetOperatorConfiguration(r.Client, logger)
+	mode, conflict, err := utils.GetOperatorConfiguration(r.Client, logger)
 	if err != nil {
 		logger.WithError(err).Error("error checking if operator is disabled")
 		return reconcile.Result{}, err
 	} else if conflict {
 		logger.Error("configuration conflict betwen legacy configmap and operator config")
 		return reconcile.Result{}, fmt.Errorf("configuration conflict")
-	} else if operatorIsDisabled {
+	} else if mode == operatorv1.CloudCredentialsModeManual {
```

Comment:
Don't need to change it, but I find the `else` distracting. There is no need for it to be an `else` since each of the previous blocks return from the function.
```
mode, conflict, err := ...
if err != nil {
   ...
}
if conflict {
   ...
}
if mode == operatatorv1.CloudCredentialsModeManual {
   ...
}
```